TextAnalytics/pitchfork_data/pitchfork_language_model.ipynb

```diff
@@ -0,0 +1,510 @@
+       "      <td>0</td>\n",
+       "      <td>4.329775</td>\n",
+       "      <td>4.249726</td>\n",
+       "      <td>0.315335</td>\n",
+       "      <td>18:34</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td>1</td>\n",
+       "      <td>4.267435</td>\n",
+       "      <td>4.192162</td>\n",
+       "      <td>0.323707</td>\n",
+       "      <td>19:00</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td>2</td>\n",
+       "      <td>4.291625</td>\n",
+       "      <td>4.180893</td>\n",
+       "      <td>0.326141</td>\n",
+       "      <td>19:05</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td>3</td>\n",
+       "      <td>4.240601</td>\n",
+       "      <td>4.156091</td>\n",
+       "      <td>0.328409</td>\n",
+       "      <td>18:44</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td>4</td>\n",
+       "      <td>4.209528</td>\n",
+       "      <td>4.138834</td>\n",
+       "      <td>0.331526</td>\n",
+       "      <td>18:41</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td>5</td>\n",
+       "      <td>4.172590</td>\n",
+       "      <td>4.115703</td>\n",
+       "      <td>0.334366</td>\n",
+       "      <td>18:42</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td>6</td>\n",
+       "      <td>4.094367</td>\n",
+       "      <td>4.093990</td>\n",
+       "      <td>0.337025</td>\n",
+       "      <td>18:41</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td>7</td>\n",
+       "      <td>4.057703</td>\n",
+       "      <td>4.073229</td>\n",
+       "      <td>0.339719</td>\n",
+       "      <td>18:42</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td>8</td>\n",
+       "      <td>4.029308</td>\n",
+       "      <td>4.055165</td>\n",
+       "      <td>0.342347</td>\n",
+       "      <td>18:42</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td>9</td>\n",
+       "      <td>3.976991</td>\n",
+       "      <td>4.040740</td>\n",
+       "      <td>0.344423</td>\n",
+       "      <td>18:42</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td>10</td>\n",
+       "      <td>3.926024</td>\n",
+       "      <td>4.033634</td>\n",
+       "      <td>0.345398</td>\n",
+       "      <td>18:42</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td>11</td>\n",
+       "      <td>3.907341</td>\n",
+       "      <td>4.032003</td>\n",
+       "      <td>0.345692</td>\n",
+       "      <td>18:42</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "#learn.fit_one_cycle(10, 2e-3, moms=(0.8,0.7), wd=0.1)\n",
+    "learn_pitchfork.fit_one_cycle(12, 2e-3/3, moms=(0.8,0.7), wd= 0.1)"
```

Comment:
again - iteratively unfreeze layers and train - track progress using something like fastprogress